### DISCUSSION TITLE
can a version be made for 1.19.4

### DISCUSSION THREAD

**Discussion by p1neC0neExploit** [vectorwing/FarmersDelight#787] [General]
### Minecraft version

1.19.2

### Farmer's Delight version

1.2

### Forge version

0.0230

### Description

not really an issue more of a request. most of my mods are 1.19.4 and i would like to add this mod to my collection but says wrong version

### Mod list

Butchers Delight

### Minimal instance

- [ ] I have tested this on a minimal instance

### Performance and shader mods

- [ ] I am using performance or shader mods

**Comment by vectorwing** [comment]
I moved this over to the Discussions tab, as it's not an issue.

The main reason I stuck with 1.19.2 is that, not only is this the most adopted 1.19 version at the moment, but because maintaining versions between .2 and .4 would be difficult moving forward.

1.19.3 and 1.19.4 both introduced a series of individual breaking changes, and 1.20 did so as well. In order to keep up a 1.19.4 build, I'd need to keep track of changes which are similar, but slightly different to 1.20, greatly adding to the labor of making FD updates.

Sorry, I'm staying on 1.19.2 only for the time being. :(
